presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -5,7 +5,9 @@ Built-in System Access Control
 A system access control plugin enforces authorization at a global level,
 before any connector level authorization. You can either use one of the built-in
 plugins in Presto, or provide your own by following the guidelines in
-:doc:`/develop/system-access-control`. Presto offers three built-in plugins:
+:doc:`/develop/system-access-control`. To run multiple system access control
+plugins at the same time see :ref:`multiple-system-access-control`. Presto
```

Comment:
Let's link to the developer page instead and remove the "Multiple System Access Control" section below. It's not really appropriate for the "Built-in System Access Control" documentation, and having a table for a single property is overkill.

(we do need higher level security docs that explain system access control, and those will be a good place to put this information in addition to the developer docs)

We could change this introduction as follows:

```
A system access control plugin enforces authorization at a global level,
before any connector level authorization. You can use one of the built-in
plugins in Presto, or provide your own by following the guidelines in
:doc:`/develop/system-access-control`.

Multiple system access control implementations may be configured at once
using the ``access-control.config-files`` configuration property. It should
contain a comma separated list of the access control property files to use
(rather than the default ``etc/access-control.properties``).

Presto offers three built-in plugins:
```